cicd/pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc

```diff
@@ -52,5 +52,4 @@ include::modules/op-annotations-for-automatic-pruning-taskruns-pipelineruns.adoc
 
 * xref:../../cicd/pipelines/authenticating-pipelines-using-git-secret.adoc#op-configuring-ssh-authentication-for-git_authenticating-pipelines-using-git-secret[Configuring SSH authentication for Git]
 * xref:../../cicd/pipelines/managing-nonversioned-and-versioned-cluster-tasks.adoc#managing-nonversioned-and-versioned-cluster-tasks[Managing non-versioned and versioned cluster tasks]
-* xref:../../cicd/pipelines/working-with-pipelines-using-the-developer-perspective.adoc#using-custom-pipeline-template-for-git-import_working-with-pipelines-using-the-developer-perspective[Using a custom pipeline template for creating and deploying an application from a Git repository]
```

Comment:
Is there a particular reason you did not add a link to the new content here instead, and just removed the old one?

Reply:
Updated in next commit